tests/topotests/lib/lutil.py

```diff
@@ -242,20 +243,27 @@ def command(self, target, command, regexp, op, result, returnJson):
             return js
         return ret
 
-    def wait(self, target, command, regexp, op, result, wait, returnJson):
-        self.log('%s:%s WAIT:%s:%s:%s:%s:%s:%s:' % \
-                 (self.l_filename, self.l_line, target, command, regexp, op, result,wait))
+    def wait(self, target, command, regexp, op, result, wait, returnJson, wait_time=0.5):
+        self.log('%s:%s WAIT:%s:%s:%s:%s:%s:%s:%s:' % \
+                 (self.l_filename, self.l_line, target, command, regexp, op, result,wait,wait_time))
         found = False
         n = 0
         startt = time.time()
-        delta = time.time() - startt
-        while delta < wait and found is False:
+
+        # Calculate the amount of `sleep`s we are going to peform.
+        wait_count = int(math.ceil(wait / wait_time)) + 1
+
+        while wait_count > 0:
+            n += 1
             found = self.command(target, command, regexp, op, result, returnJson)
-            n+=1
-            delta = time.time() - startt
-            self.log('\tFound: %s n: %s delta: %s and wait: %s' % (found, n, delta, wait))
-            if delta < wait and found is False:
-                time.sleep (0.5)
+            if found is not False:
```

Comment:
how about `if found:`

Reply:
on second thought lets just get ci working again lol

Reply:
> how about if found:

I actually spend a while here until I figured out that this functions returns `False` on failure and anything else on success. Some types might be interpreted as false... so in order to avoid that spectrum we have to look for `not False`.